cluster-autoscaler/cloudprovider/clusterapi/clusterapi_provider.go

```diff
@@ -142,22 +146,32 @@ func BuildClusterAPI(opts config.AutoscalingOptions, do cloudprovider.NodeGroupD
 	}
 
 	// Grab a dynamic interface that we can create informers from
-	dc, err := dynamic.NewForConfig(externalConfig)
+	managementClient, err := dynamic.NewForConfig(externalConfig)
```

Comment:
i thought we were separating this work into the follow up commit?

Reply:
In this instance I just renamed the variable to be more descriptive of it's use, there is no separation between kubeconfigs that are used by the clients with this change.

Reply:
ok, cool. i figured that was the case but wanted to be sure.